tests/testsuite/namespace.go

```diff
@@ -307,10 +307,6 @@ func detectInstallNamespace() {
 func GetLabelsForNamespace(namespace string) map[string]string {
 	labels := map[string]string{
 		cleanup.TestLabelForNamespace(namespace):         "",
-		"security.openshift.io/scc.podSecurityLabelSync": "false",
-	}
-	if namespace == NamespacePrivileged {
```

Comment:
Why do you want to drop this?

Reply:
It's a direct consequence of the previous line:
directly setting `pod-security.kubernetes.io/enforce=privileged` on makes sense on OCP/OKD only if `security.openshift.io/scc.podSecurityLabelSync=false` is also set, otherwise it will override it.

Reply:
Sure but I would still like to be able to test vanilla Kubernetes.

Reply:
so? are you suggesting to remove only `security.openshift.io/scc.podSecurityLabelSync=false` and try applying an external workaround on OCP/OKD?

Reply:
amended

Reply:
Yes, this namespace is used only for test-related pods so it should be ok.